### PR TITLE
Add sound and notification when a roamer pokemon appears

### PR DESCRIPTION
**Pull request by carlos93**
Minor improvement.

Just notify the player that a roamer pokemon has just appeared.

The new .mp3 file is just a copy of the shiny sound. I can change it if needed.

### CODE REVIEW

File: src/scripts/pokemons/PokemonFactory.ts [hunk 1]
```diff
@@ -50,6 +51,14 @@ class PokemonFactory {
             LogEvent('encountered shiny', 'shiny pokemon', 'wild encounter',
                 Math.floor(App.game.statistics.totalPokemonEncountered() / App.game.statistics.totalShinyPokemonEncountered()));
         }
+        if (roamer) {
+            Notifier.notify({
+                message: `🌠 You encountered a roamer ${name}! 🌠`,
```
Comment (Jaaslet) [inline src/scripts/pokemons/PokemonFactory.ts:56]: It's called "Roaming <pokemonName>". This change also needs to be applied a couple of other places.

File: docs/scripts/script.min.js [hunk 2]
```diff
@@ -21509,7 +21509,8 @@ class PokemonFactory {
             return new BattlePokemon('MissingNo.', 0, PokemonType.None, PokemonType.None, 0, 0, 0, 0, new Amount(0, GameConstants.Currency.money), false, 0);
         }
         let name;
-        if (PokemonFactory.roamingEncounter(route, region)) {
+        const roamer = PokemonFactory.roamingEncounter(route, region)
+        if (roamer) {
```
Comment (Jaaslet) [inline docs/scripts/script.min.js:21513]: Don't submit anything in the docs-folder. The buildscript will be run for each release.

**Review by Jaaslet — CHANGES_REQUESTED** [review]
Looks good and works fine. Just have some small changes

File: src/scripts/pokemons/PokemonFactory.ts (hunk 1 above)
Comment (carlos93) [reply src/scripts/pokemons/PokemonFactory.ts:56]: Sure, I can remove the emojis and change roamer -> roaming

File: docs/scripts/script.min.js (hunk 2 above)
Comment (carlos93) [reply docs/scripts/script.min.js:21513]: I'll revert this

**Comment by Jaaslet** [comment]
The check is failing. There is a missing semicolon

**Comment by carlos93** [comment]
Done

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Comment by carlos93** [comment]
Thanks!

**Pull request merged**